chpl/chpl-resources/src/main/resources-qa/log4j2.xml

```diff
@@ -523,9 +422,5 @@
             additivity="false">
```

Comment:
DO these three jobs exist?

Reply:
They all get removed in a different ticket